apps/storybook/src/Tabs.stories.tsx

```diff
@@ -206,68 +259,56 @@ export const HorizontalOverflow = () => {
         padding: 8,
       }}
     >
-      <Tabs
-        type='default'
-        labels={labels}
-        overflowOptions={{ useOverflow: true }}
-        onTabSelected={setActiveIndex}
-        activeIndex={activeIndex}
-        actions={[<Button key={'button'}>Button</Button>]}
-      >
-        {getContent()}
-      </Tabs>
+      <Tabs.Wrapper value={active} onValueChange={setActive}>
+        <Tabs.TabList>
+          {tabData?.map((item) => {
+            return (
+              <Tabs.Tab
+                key={item.name}
+                value={item.name}
+                disabled={item.disabled}
+                label={item.name}
+              />
+            );
+          })}
+        </Tabs.TabList>
+
+        <Tabs.Actions>
+          <Button key={'button'}>Button</Button>
+        </Tabs.Actions>
+
+        {tabData.map((item) => (
+          <Tabs.Panel key={item.name} value={item.name}>
+            {item.content}
+          </Tabs.Panel>
+        ))}
+      </Tabs.Wrapper>
     </div>
   );
 };
 
 export const VerticalOverflow = () => {
```

Comment:
May not be related to this PR, but just noticed some shift in the width of the tabs panel in the "Vertical Overflow" story

[14.webm](https://github.com/iTwin/iTwinUI/assets/45748283/2bb1bd2a-facc-408b-bb07-a4b5e5ffbda0)

Reply:
i cannot reproduce?

Reply:
I am on a Windows 11 + Edge 117. What I did was:
* Press tab to focus the selected "Item 11`"
* Press up or down

Reply:
Could not find a reason why this is happening.

Reply:
Maybe we can make another issue for this since it is rare. If it happens again, we can update that issue.